SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -178,6 +179,9 @@ public static void Main(string[] args)
 
                 var windowsNoEditorPath = Path.Combine(stagingDir, "WindowsNoEditor");
 
+                // force spatial networking to be enabled
```

Comment:
Comment not necessary now since it's pretty much the name of the function, can remove the others too.